build.gradle

```diff
@@ -58,12 +58,14 @@ tasks.withType(KotlinCompile).configureEach {
 }
 
 dependencies {
-    api platform('com.exactpro.th2:bom:4.2.0')
+    api platform('com.exactpro.th2:bom:4.4.0-update-libs-5345334944-c71fdc9-SNAPSHOT')
+    implementation "com.exactpro.th2:common:5.3.0-separate-executor-5391854885-afab720-SNAPSHOT"
```

Comment:
Please, leave a TODO that this should not be here... We should depend only on the grpc and transport